app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -359,6 +362,13 @@ protected void setIsReply(boolean isReply) {
 
     }
 
+    private void setReplyCount(int repliesCount) {
+        // This label only exists in the non-detailed view (to match the web ui)
+        if (replyCountLabel != null) {
+            replyCountLabel.setText((repliesCount > 1 ? replyCountLabel.getContext().getString(R.string.status_count_one_plus) : NUMBER_FORMAT.format(repliesCount)));
```

Comment:
Is there really a need to format a 0? Like, how could it be displayed differently?